src/lib/helpers/leave.ts

```diff
@@ -18,24 +18,100 @@
  *
  */
 
-import {MyLeave} from 'store/leave/leave-usage/types';
+import {MyLeave, Leave, LeaveStatus} from 'store/leave/leave-usage/types';
 
 const LEAVE_TYPE_COLORS = [
   '#445abf',
   '#4ece68',
   '#c15cb8',
   '#982727',
   '#db1580',
+  '#037b20',
+  '#0288d9',
+  '#d76900',
+  '#732673',
+  '#405040',
 ];
 
+const LEAVE_STATUS_MAP = {
+  REJECTED: 'Rejected',
+  CANCELLED: 'Cancelled',
+  'PENDING APPROVAL': 'Pending Approval',
```

Comment:
what if const as PENDING_APPROVAL ?

Reply:
Those const value coming from OS instance. Here I only maintain that mapping